### PR TITLE
daos-6151 documentation: Refresh Online Documentations

### PR DESCRIPTION
**Pull request by cdurf1**
refreshed online docs

Skip-build: true
Skip-test: true

Signed-off-by: craig <craig.durfey@intel.com>

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+[1120/074939.454:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1120/075445.210:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1120/111552.436:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1120/111927.547:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1120/112413.874:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1120/112841.780:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1124/134809.797:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1125/072854.644:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1125/073646.713:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
+[1125/092357.220:ERROR:registration_protocol_win.cc(131)] TransactNamedPipe: The pipe has been ended. (0x6D)
```
Comment (johannlombardi) [inline debug.log:23]: Should not be checked in.

File: doc/js/version-select.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    var container = document.createElement("div");
+    container.id = "version-selector";
+    container.className = "md-nav__item";
+    container.appendChild(select);
+
+    var sidebar = document.querySelector(".md-nav--primary > .md-nav__list");
+    sidebar.parentNode.insertBefore(container, sidebar);
+  };
+  xhr.send();
+});
```
Comment (johannlombardi) [inline doc/js/version-select.js:49]: Is that really required to be in the tree?

**Review by johannlombardi — CHANGES_REQUESTED** [review]
Please remove the unneeded files. Thanks.

File: doc/js/version-select.js (hunk 2 above)
Comment (cdurf1) [reply doc/js/version-select.js]: deleted file

**Pull request closed without merging**